src/components/requests/Requests.css

```diff
@@ -0,0 +1,121 @@
+.requests {
+  box-sizing: border-box;
+  display: flex;
+  text-align: center;
+  flex-direction: column;
+  justify-content: center;
+  padding: 100px 80px;
+}
+
+.requests__title {
+  margin: 0 auto;
+  max-width: 725px;
+  font-size: 48px;
+  font-weight: 400;
+}
+
+.requests__subtitle {
+  color: #616161;
+  max-width: 825px;
+  font-size: 30px;
+  font-weight: 400;
+  margin: 40px auto 70px auto;
+}
+
+.requests__card-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-column-gap: 20px;
+  justify-items: center;
+  align-self: center;
+}
+
+.card {
+  text-align: center;
+  box-shadow: rgba(99, 99, 99, 0.2) 0px 2px 8px 0px;
+  border-radius: 10px;
+  transition: all 0.3s ease;
+  cursor: pointer;
+}
+
+.card:hover {
+  box-shadow: rgba(149, 157, 165, 0.2) 0px 8px 24px;
+  transform: translate3D(0, -1px, 0) scale(1.02);
+}
+
+.card__hashtag {
+  color: #ff9900;
+  margin-top: 30px;
+  margin-bottom: 10px;
+}
+
+.card__title {
+  font-size: 30px;
+  line-height: 40px;
+  margin-bottom: 20px;
+}
+
+.card__image {
+  width: 95%;
+  height: auto;
+}
+
+@media screen and (max-width: 1024px) {
+  .requests {
+    padding-left: 40px;
+    padding-right: 40px;
+  }
+
+  .requests__card-container {
+    grid-template-columns: 1fr 1fr;
+    grid-row-gap: 40px;
+    grid-column-gap: 40px;
+    grid-template-areas:
```

Comment:
This is the ticket for the other sections @colin was working on, using GTA's instead of flexbox.